client/state/preferences/reducer.js

```diff
@@ -50,7 +50,7 @@ export const localValues = ( state = {}, action ) => {
 export const remoteValues = withSchemaValidation( remoteValuesSchema, ( state = null, action ) => {
 	switch ( action.type ) {
 		case PREFERENCES_RECEIVE: {
-			const { values } = action;
+			const { values = {} } = action;
```

Comment:
Should this be a constant outside the reducer? Or could be instead return the previous state?

Reply:
Alternatively we could also `return values ?? {}` on the next line.

Reply:
Return the previous state: the previous state is probably `null` which means "not loaded yet". But we have loaded the preferences and they are empty. If we kept `null` the state wouldn't be telling the truth.

Constant outside reducer: yes, technically that's better but we're usually loading the preferences just once, so an inline value is also fine.